app/presenters/iiif_print/iiif_manifest_presenter_behavior.rb

```diff
@@ -25,5 +25,74 @@ def sequence_rendering
           'label' => I18n.t("hyrax.manifest.download_text") + (rendering.label || '') }
       end.flatten
     end
+
+    module DisplayImagePresenterBehavior
```

Comment:
this should have an "OVERIDE: line at the top of the file, since it overrides specific Hyrax functionality. Also, we probably need a line in the README that covers it.

From the conversation we had with ESSI this morning, i think we might want to rename `SERVERLESS_IIIF_URL` to `EXTERNAL_IIIF_URL` or something else that implies that value can be either serverless_iiif or direct cantaloup iiif

Reply:
updated ✅